content/posts/dariah-de-publikator-video-tutorial/index.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: DARIAH-DE Publikator Video Tutorial
+lang: en
```

Comment:
unsure about this: should the `lang` refer to the language of the external content (in this case, the linked youtube video), or to the language of the resource description on DC?

Reply:
I would think that this would refer to the language of the main resource itself rather than the description on DC.  We have only used 'en' up to now, though, as we didn't have a way of searching for non-English items.  We also didn't have enough non-English items!

Do we now have the option to search for items by language?

Reply:
> Do we now have the option to search for items by language?

no - every item is indexed regardless of language setting, however we currently don't have a way to specifically search for or show resources by language. do we want to add this?

also, the CMS config currently only allows setting "en" as resource language (see https://github.com/DARIAH-ERIC/dariah-campus/blob/main/src/cms/collections/posts.collection.ts#L49-L54) - if we want to open this up, we should decide if we want to preselect a list of supported languages, or if `lang` should become a plain text input field.

Reply:
we can't search items by language yet. this is not an ideal situation —  i would expect german videos to have german language descriptions etc. but it doesn't make sense to have only one german resource.

so, I'd say, keep "en" here, but this should really be the exception. let's stick to english items until we have a proper internationalization strategy.